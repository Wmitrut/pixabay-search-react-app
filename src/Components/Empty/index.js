import Sunflower from '../../Assets/sunflower.jpeg';

const Empty = () => {
  return (
    <div className="content-center justify-center text-center">
      <img
        src={Sunflower}
        alt="Sunflower field"
        className="m-auto my-2 h-64 w-96 rounded-lg object-cover"
      />
      <span className="text-gray-500">
        Type away in the search bar to find beautiful flowers and more...
      </span>
    </div>
  );
};

export default Empty;
