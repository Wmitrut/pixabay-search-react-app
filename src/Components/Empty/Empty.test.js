import renderer from 'react-test-renderer';

import Empty from './index';

it('Renders correctly without data', () => {
  const tree = renderer.create(<Empty />).toJSON();
  expect(tree).toMatchSnapshot();
});
