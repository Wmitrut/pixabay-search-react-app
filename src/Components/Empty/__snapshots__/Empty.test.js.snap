// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly without data 1`] = `
<div
  className="content-center justify-center text-center"
>
  <img
    alt="Sunflower field"
    className="m-auto my-2 h-64 w-96 rounded-lg object-cover"
    src="sunflower.jpeg"
  />
  <span
    className="text-gray-500"
  >
    Type away in the search bar to find beautiful flowers and more...
  </span>
</div>
`;
